assets/js/vue-apps/components/budget-input.vue

```diff
@@ -43,102 +42,93 @@ export default {
         },
-        shouldCreateNewRow: function() {
+        shouldCreateNewRow() {
             const lastItem = this.budgetRows[this.budgetRows.length - 1];
             return (
                 lastItem &&
                 this.budgetRows.length < this.maxItems &&
                 (!isEmpty(lastItem.item) || !isEmpty(lastItem.cost))
             );
         },
-        getLineItemName: function(index, subFieldName) {
+        getLineItemName(index, subFieldName) {
             return `${this.fieldName}[${index}][${subFieldName}]`;
+        },
+        canDelete(index) {
+            return this.budgetRows.length > 1 && index !== this.budgetRows.length - 1;
         }
     }
 };
 </script>
 
-<!-- If we update this we should change the Vue component too -->
+<!-- If we update this we should change the Nunjucks component too -->
 <template>
     <div class="ff-budget">
-        <table class="ff-budget__table">
-            <thead>
-                <tr>
-                    <th scope="col" class="ff-budget__item-col">Item or activity</th>
-                    <th scope="col">Amount from us</th>
-                    <th scope="col" class="ff-budget__actions"></th>
-                </tr>
-            </thead>
-            <tbody>
-                <tr
-                    v-for="(lineItem, index) in budgetRows"
-                    :aria-live="lineItem.isNew ? 'polite' : false"
-                    :key="index"
-                    v-cloak
-                >
-                    <td>
-                        <input
-                            type="text"
-                            autocomplete="off"
-                            :name="getLineItemName(index, 'item')"
-                            v-model="lineItem.item"
-                            placeholder="Add item name"
-                            class="ff-text u-block-full"
-                        />
-                    </td>
-                    <td>
-                        <div class="ff-field-prepend">£</div>
+        <ol class="ff-budget__list">
+            <li class="ff-budget__row" v-for="(lineItem, index) in budgetRows" :key="index">
+                <div class="ff-budget__row-item">
+                    <label class="ff-label" :for="getLineItemName(index, 'item')">
+                        Item or activity
+                    </label>
+                    <input
+                        class="ff-text u-block-full"
+                        type="text"
+                        :name="getLineItemName(index, 'item')"
+                        :id="getLineItemName(index, 'item')"
+                        autocomplete="off"
+                        v-model="lineItem.item"
+                    />
+                </div>
+                <div class="ff-budget__row-amount">
+                    <label class="ff-label" :for="getLineItemName(index, 'cost')">
+                        Amount
+                    </label>
+                    <div class="ff-currency">
```

Comment:
This is a new `ff-currency` field, fixes some cross browser issues with the previous approach. There's a future job to update other instances of this to use the new markup. For now limiting it to the budget input.